# Telomere-to-telomere consortium CHM13 project

We have sequenced the CHM13hTERT human cell line with a number of technologies.
 Human genomic DNA was extracted from the cultured cell line.
 As the DNA is native, modified bases will be preserved.
 The data includes 30x [PacBio](https://www.pacb.com/) [HiFi](https://www.ncbi.nlm.nih.gov/sra/?term=SRX789768*+CHM13),
 120x coverage of [Oxford Nanopore](https://nanoporetech.com), 70x [PacBio](https://www.pacb.com/) CLR, 50x [10X Genomics](https://www.10xgenomics.com),
 as well as [BioNano DLS](https://bionanogenomics.com/technology/dls-technology/) and [Arima Genomics HiC](https://arimagenomics.com/kit/).
 Most raw data is available from this site,
 with the exception of the PacBio data which was generated by the University of Washington/PacBio and is available from [NCBI SRA](https://www.ncbi.nlm.nih.gov/sra?linkname=bioproject_sra_all&from_uid=269593).

A UCSC browser is available for [v2.0](https://genome.ucsc.edu/h/GCA_009914755.4)
 (as well as legacy [v1.0](http://genome.ucsc.edu/cgi-bin/hgTracks?genome=t2t-chm13-v1.0&hubUrl=http://t2t.gi.ucsc.edu/chm13/hub/hub.txt)
 and [v1.1](http://genome.ucsc.edu/cgi-bin/hgTracks?genome=t2t-chm13-v1.1&hubUrl=http://t2t.gi.ucsc.edu/chm13/hub/hub.txt) versions).
 An interactive dotplot visualization of all genomic repeats is also available from [resgen.io](https://resgen.io/paper-data/T2T-Nurk-et-al-2021/views/t2t-identity-v2).
 Known issues identified in the assembly are tracked at [CHM13 issues](https://github.com/marbl/CHM13-issues).

## Latest assembly release

### T2T-CHM13v2.0
[Complete T2T reconstruction of a human genome with Y](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/analysis_set/chm13v2.0.fa.gz).
Changes from v1.1 is the addition of a finished chromosome Y from the [GIAB](https://www.nist.gov/programs-projects/genome-bottle) HG002/NA24385 sample,
 sequenced both by [GIAB](https://github.com/genome-in-a-bottle/giab_data_indexes) and [HPRC](https://humanpangenome.org/hg002/).
 This genome is also available at [NCBI (GCA_009914755.4)](https://www.ncbi.nlm.nih.gov/assembly/GCA_009914755.4) and at [UCSC](https://genome.ucsc.edu/h/GCA_009914755.4).
 Note that even though the UCSC browser shows the Genbank accessions as sequence names on the browser itself, it can load annotations in BED/bigBed/BAM/CRAM/bigWig and other formats or search using the "chr1/2/etc" names.

Previous assembly releases are available below:
 - [T2T-CHM13](Previous_assembly_release_CHM13.md): v0.7-v1.1
 - [T2T-HG002XY](Previous_assembly_release_HG002.md): v0.7-2.7

## Downloads

### Sequencing data
The sequencing dataset generated for CHM13 has been moved to [this page](Sequencing_data.md).

### Analysis set
Analysis set for using T2T-CHM13v2.0 as a reference for mapping based research is available at [aws](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/CHM13/assemblies/analysis_set/)
 with a [README](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/analysis_set/README.txt).
 - [chm13v2.0.fa.gz](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/analysis_set/chm13v2.0.fa.gz):
  T2T-CHM13v2.0 assembly with sequences soft-masked using the repeat models discovered by the T2T team.
   The original sequence accession numbers are shown in the FASTA header.
 - [chm13v2.0_noY.fa.gz](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/analysis_set/chm13v2.0_noY.fa.gz):
  excluding the Y chromosome. This file only contains sequences derived from the CHM13 cell line and is identical to T2T-CHM13v1.1.
  Use this file for benchmarking assemblies of CHM13.
 - [chm13v2.0_PAR.bed](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/analysis_set/chm13v2.0_PAR.bed): pseudoautosomal regions (PARs)
 - [chm13v2.0_maskedY.fa.gz](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/analysis_set/chm13v2.0_maskedY.fa.gz): PARs on chrY hard masked to "N"
 - [chm13v2.0_maskedY.rCRS.fa.gz](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/analysis_set/chm13v2.0_maskedY_rCRS.fa.gz):
  PARs on chrY hard masked to "N" and mitochodrion replaced with rCRS (AC:NC_012920.1)

> Sep. 28 2022 update: all analysis-set fa.gz files have been re-compressed with bgzip.
> Index files are available at [aws](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/CHM13/assemblies/analysis_set/)
> with updated md5s in the [README](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/analysis_set/README.txt).

### Gene annotation
 - [JHU RefSeqv110 + Liftoff v4](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_RefSeq_Liftoff_v4.gff3): 
 This contains _curated_ annotations of the ampliconic genes on the Y chromosome, correcting annotation errors in GENCODEv35 CAT/Liftoff and RefSeqv110 annotation.
 Additional copies found in T2T-Y were annotated to the closest available gene in RefSeq, allowing multiple genes to have the same common name.
 This file has been modified to correct special character issues from the original file. More description is available [here](https://ccb.jhu.edu/T2T.shtml).
 - [UCSC GENCODEv35 CAT/Liftoff v2](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13.draft_v2.0.gene_annotation.gff3)
   - CAT/Liftoff v1 annotation for VEP in [Sorted GFF](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_GENCODEv35_CAT_Liftoff.vep.gff3.gz) and [TABIX index](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_GENCODEv35_CAT_Liftoff.vep.gff3.gz.tbi)
   - [Protein coding translated transcripts](https://hgdownload.soe.ucsc.edu/hubs/GCA/009/914/755/GCA_009914755.4/genes/catLiftOffGenesV1.protein.fa.gz) from CAT/Liftoff v1 annotation.
   Note, these are transcripts not genes and only searchable by transcript ID (IDs like LOFF_T not LOFF_G).
 - [NCBI RefSeqv110](https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/009/914/755/GCF_009914755.1_T2T-CHM13v2.0/GCF_009914755.1_T2T-CHM13v2.0_genomic.gtf.gz) from [FTP](https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/009/914/755/GCF_009914755.1_T2T-CHM13v2.0/)
 - [EBI GENCODEv38 r2](https://ftp.ensembl.org/pub/rapid-release/species/Homo_sapiens/GCA_009914755.4/geneset/2022_07/Homo_sapiens-GCA_009914755.4-2022_07-genes.gff3.gz) from [HPRC Projects](https://projects.ensembl.org/hprc/)

### Repeat annotation
 - [Cytobands](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_cytobands_allchrs.bed)
 - Segmental Duplications, v2022-03-11 in [simple](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_SD.bed) and [full](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_SD.full.bed) bed format
 - [Cen/Sat v2.0](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_censat_v2.0.bed):
  A more comprehensive centromere/satellite repeat annotation.
 - RepeatMasker v4.1.2p1.2022Apr14 in [bed](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_RepeatMasker_4.1.2p1.2022Apr14.bed) or [native out](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_RepeatMasker_4.1.2p1.2022Apr14.out)
 - [Composite Repeats, 2022DEC](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_composite-repeats_2022DEC.bed)
 - [New Satellites, 2022DEC](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_new-satellites_2022DEC.bed)
 - [chrXY sequence class, v1](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_chrXY_sequence_class_v1.bed)
 - [Telomere](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_telomere.bed)
 - Y specific annotation
   - [Palindromes and Inverted Repeats, v1](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0Y_inverted_repeats_v1.bed)
   - [Amplicons v1](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0Y_amplicons_v1.bed)
   - [AZFa, AZFb, AZFc and DYZ v1](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0Y_AZF_DYZ_v1.bed)

### Epigenetic profile
 - [ENCODE](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/CHM13/assemblies/annotation/regulation/ENCODE/), recalled on T2T-CHM13v2.0
 - [HG002 5mC CpG and other methylation from ONT and HiFi](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/CHM13/assemblies/annotation/regulation/)

### Variant calls
  - [1000 Genomes Project](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/CHM13/assemblies/variants/1000_Genomes_Project/chm13v2.0/), recalled on T2T-CHM13v2.0. Currently available only for chr X and Y, for the entire 3,202 samples or the unrelated 2504 samples.
  - [Simons Genome Diversity Project](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/CHM13/assemblies/variants/SGDP/chm13v2.0/), recalled on T2T-CHM13v2.0.
  - [gnomAD v3.1.2](https://ftp.ensembl.org/pub/rapid-release/species/Homo_sapiens/GCA_009914755.4/ensembl/variation/2022_10/vcf/Homo_sapiens-GCA_009914755.4-2022_10-gnomad.vcf.gz) from [FTP](https://ftp.ensembl.org/pub/rapid-release/species/Homo_sapiens/GCA_009914755.4/ensembl/variation/2022_10/vcf/):
  This is a lifted over version from GRCh38, annotated with predicted molecular consequence and transcript-specific variant deleteriousness scores from PolyPhen-2 and SIFT using Ensembl Variant Effect Predictor.
  - [ClinVar 20220313](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/liftover/chm13v2.0_ClinVar20220313.vcf.gz), lifted over from GRCh38. See [description](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/liftover/ClinVar.html)
  - [GWAS v1.0](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/liftover/chm13v2.0_GWASv1.0rsids_e100_r2022-03-08.vcf.gz), lifted over from GRCh38. See [description](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/liftover/GWAS.html)
  - [dbSNP build 155](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/liftover/chm13v2.0_dbSNPv155.vcf.gz), lifted over from GRCh38. See [description](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/liftover/dbSNP.html)
  - [Variants disappearing in GRCh38-Y coordinates, v0.005](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/variants/GRCh38/variants_disappearing/20220324_v0.005_HG002XYvsGRCh38/preliminary-draft-small-variant-benchmark-files/GRCh38_T2T-XY-v2.7_dipcall-z2k.vcf.gz) when using T2T-Y as a reference, more details are [here](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/CHM13/assemblies/variants/GRCh38/variants_disappearing/20220324_v0.005_HG002XYvsGRCh38/).

### Liftover resources
 - 1:1 Liftover GRCh38 <-> T2T-CHM13v2.0, see [description](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/chain/v1_nflo/v1_nflo_description.html)
   - GRCh38  -> T2T-CHM13v2.0: [grch38-chm13v2.chain](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/chain/v1_nflo/grch38-chm13v2.chain)
   - GRCh38 <-  T2T-CHM13v2.0: [chm13v2-grch38.chain](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/chain/v1_nflo/chm13v2-grch38.chain)
   - Alignment [grch38-chm13v2.paf](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/chain/v1_nflo/grch38-chm13v2.paf)
 - 1:1 Liftover hg19 <-> T2T-CHM13v2.0
   - hg19  -> T2T-CHM13v2.0[hg19-chm13v2.chain](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/chain/v1_nflo/hg19-chm13v2.chain)
   - hg19 <-  T2T-CHM13v2.0[chm13v2-hg19.chain](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/chain/v1_nflo/chm13v2-hg19.chain)
   - Alignment [hg19-chm13v2.paf](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/chain/v1_nflo/hg19-chm13v2.paf)

## Notes on downloading files

Files are generously hosted by Amazon Web Services under s3://human-pangenomics/T2T/CHM13 and through this [web interface](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/CHM13).

Although available as straight-forward HTTP links, download performance is improved by using the Amazon Web Services [command-line interface](https://aws.amazon.com/cli/).
 References should be amended to use the `s3://` addressing scheme, i.e. replace `https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/` with `s3://human-pangenomics/T2T` to download.
 For example, to download `CHM13_prep5_S13_L002_I1_001.fastq.gz` to the current working directory use the following command.

    aws s3 --no-sign-request cp s3://human-pangenomics/T2T/CHM13/10x/CHM13_prep5_S13_L002_I1_001.fastq.gz .

or to download the full dataset use the following command.

    aws s3 --no-sign-request sync s3://human-pangenomics/T2T/CHM13/ .

The s3 command can also be used to get information on the dataset, for example reporting the size of every file in human-readable format.

    aws s3 --no-sign-request ls --recursive --human-readable --summarize s3://human-pangenomics/T2T/CHM13/ 

or to obtain technology-specific sizes.

    aws s3 --no-sign-request ls --recursive --human-readable --summarize s3://human-pangenomics/T2T/CHM13/nanopore/fast5
    aws s3 --no-sign-request ls --recursive --human-readable --summarize s3://human-pangenomics/T2T/CHM13/nanopore/rel2
    aws s3 --no-sign-request ls --recursive --human-readable --summarize s3://human-pangenomics/T2T/CHM13/assemblies

Amending the `max_concurrent_requests` etc. settings as per [this guide](https://docs.aws.amazon.com/cli/latest/topic/s3-config.html) will improve download performance further.

## Contact

Please raise issues on this Github repository concerning this dataset.

## Data reuse and license
All data is released to the public domain ([CC0](https://creativecommons.org/publicdomain/zero/1.0/)) and we encourage its reuse.
 We would appreciate if you would acknowledge and cite the "Telomere-to-Telomere" (T2T) Consortium for the creation of this data.
 More information about our consortium can be found on the [T2T homepage](https://sites.google.com/ucsc.edu/t2tworkinggroup/) and a list of related citations is available below:

### T2T-CHM13v2.0, datasets released along the v2.0 and the T2T-Y chromosome
* Rhie A, Nurk S, Cechova M, Hoyt SJ, Taylor DJ, et al. [The complete sequence of a human Y chromosome](https://doi.org/10.1101/2022.12.01.518724). bioRxiv, 2022.

### The complete sequence of a human genome and companion papers (T2T-CHM13v0.9-v1.1):
1. Nurk S, Koren S, Rhie A, Rautiainen M, et al. [The complete sequence of a human genome](https://doi.org/10.1126/science.abj6987). Science, 2022.
2. Vollger MR, et al. [Segmental duplications and their variation in a complete human genome](https://doi.org/10.1126/science.abj6965). Science, 2022.
3. Gershman A, et al. [Epigenetic Patterns in a Complete Human Genome](https://doi.org/10.1126/science.abj5089). Science, 2022.
4. Aganezov S, Yan SM, Soto DC, Kirsche M, Zarate S, et al. [A complete reference genome improves analysis of human genetic variation](https://doi.org/10.1126/science.abl3533). Science, 2022.
5. Hoyt SJ, et al. [From telomere to telomere: the transcriptional and epigenetic state of human repeat elements](https://doi.org/10.1126/science.abk3112). Science, 2022.
6. Altemose N, et al. [Complete genomic and epigenetic maps of human centromeres](https://doi.org/10.1126/science.abl4178). Science, 2022.
7. Wagner J, et al. [Curated variation benchmarks for challenging medically relevant autosomal genes](https://doi.org/10.1038/s41587-021-01158-1). Nat Biotechnol, 2022.
8. McCartney AM, Shafin K, Alonge M, et al. [Chasing perfection: validation and polishing strategies for telomere-to-telomere genome assemblies](https://doi.org/10.1038/s41592-022-01440-3). Nat Methods, 2022.
9. Formenti G, Rhie A, et al. [Merfin: improved variant filtering, assembly evaluation and polishing via k-mer validation](https://doi.org/10.1101/2021.07.16.452324). Nat Methods, 2022.
10. Jain C, et al. [Long-read mapping to repetitive reference sequences using Winnowmap2](https://doi.org/10.1038/s41592-022-01457-8). Nat Methods, 2022.
11. Altemose N, Maslan A, Smith OK et al. [DiMeLo-seq: a long-read, single-molecule method for mapping proteinâ€“DNA interactions genome wide](https://doi.org/10.1038/s41592-022-01475-6). Nat Methods, 2022.

### Earlier citations:
1. Vollger MR, et al. [Improved assembly and variant detection of a haploid human genome using single-molecule, high-fidelity long reads](https://doi.org/10.1111/ahg.12364). Annals of Human Genetics, 2019.
2. Miga KH, Koren S, et al. [Telomere-to-telomere assembly of a complete human X chromosome](https://dx.doi.org/10.1038/s41586-020-2547-7). Nature, 2020.
3. Nurk S, Walenz BP, et al. [HiCanu: accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads](https://doi.org/10.1101/gr.263566.120). Genome Research, 2020.
4. Logsdon GA, et al. [The structure, function, and evolution of a complete human chromosome 8](https://doi.org/10.1038/s41586-021-03420-7). Nature, 2021.

## History

    * rel1 and 2: 2nd March 2019. Initial release.
    * asm v0.6 and canu rel2 assembly: 28th May 2019. Assembly update.
    * Hi-C data added: 25th July 2019. Data update.
    * asm v0.6 alignments of rel2 added: 30th Aug 2019. Data Update
    * rel3: 16th Sept 2019. Data update.
    * chrX v0.7, canu 1.9 and flye 2.5 rel3 assembly: 24th Oct 2019. Assembly update.
    * shasta rel3 assembly: 20th Dec 2019. Assembly update.
    * chr8 v3, rel4 data: 21 Feb 2020. Data and assembly update.
    * update rel3 partition names since some tars included more than a single partition. 16 Apr 2020.
    * add CLR/HiFi mappings to chrX v0.7. 8 May 2020.
    * update partitions 23,28,30,53,55 and add 227-231 (data was missing from upload). 13 May 2020. Data update.
    * add rel5 guppy 3.6.0 data: 4 Jun 2020. Data update.
    * add chr8 v9. Aug 26 2020. Assembly update.
    * add v0.9/v1.0 genome releases. Sept 22 2020. Assembly update.
    * add v0.9/v1.0 alignment files. Sept 29 2020. Assembly update.
    * add new UW data. Oct 6 2020. Data update.
    * add rna-seq data. Dec 4 2020. Data update.
    * add repeat and telomere annotations for v1.0. Dec 17 2020. Assembly annotation update.
    * v1.1 assembly and related files. May 7 2021. Assembly update.
    * v2.0 assembly and related files. Dec 2 2022. Assembly and annotation update.
